File: code_java/Vectores.java
import java.util.Scanner;

public class Vectores {
    public static void intercambiar(int n) {
        int[] vector = new int[n];
        Scanner scanner = new Scanner(System.in);

        System.out.println();
        for (int i = 0; i < n; i++) {
            System.out.print("Escriba el valor " + (i + 1) + " : ");
            vector[i] = scanner.nextInt();
        }

        System.out.println("\nArreglo original");
        for (int j = 0; j < n; j++) {
            System.out.print(vector[j] + " ");
        }

        System.out.println("\nArreglo resultante");
        for (int j = n - 1; j >= 0; j--) {
            System.out.print(vector[j] + " ");
        }
    }

    public static boolean primo(int p) {
        int flag = 0;

        for (int i = 1; i <= p; i++) {
            if (p % i == 0) {
                flag += 1;
            }
        }

        return flag == 2;
    }
}

File: code_c/Vectores.c
#include <stdio.h>

int Vectores(int n){
    
    int vector[n];
    int i;
    int j;
    
    printf("\n");
    for(i = 0; i < n; i++){
        printf("Escriba el valor %d : ", i+1);
        scanf("%d", &vector[i]);
    }
    
    printf("\nArreglo original\n");
    for(j = 0; j < n; j++){
        printf("%d ", vector[j]);
    }
    
    printf("\nArreglo resultante\n");
    for(j = n-1; j >= 0; j--){
        printf("%d ", vector[j]);
    }
    
    return 0;
}